aspnetcore/performance/caching/response.md

```diff
@@ -39,7 +39,7 @@ The primary HTTP header used for caching is `Cache-Control`. The [HTTP 1.1 speci
    Indicates the response **must not** be used by a cache to satisfy any subsequent request (without successful revalidation with the origin server).
 
 > [!NOTE]
-> **Response caching does not cache responses on the web server**. It differs from [output caching](http://www.asp.net/mvc/overview/older-versions-1/controllers-and-routing/improving-performance-with-output-caching-cs), which would cache responses in memory on the server in earlier versions of ASP.NET and ASP.NET MVC. Output caching middleware is planned to be added to ASP.NET Core in a future release.
+> Caching responses on the web server can be enabled by adding the [response caching middleware](https://github.com/aspnet/responsecaching).
```

Comment:
The link should probably point to the response caching middleware docs once available.